### PR TITLE
Update buddy.js

### PR DESCRIPTION
**Pull request by SirPhoenix88**
Added ability spam, soulbreak spam.

### CODE REVIEW

**Comment by ThauEx** [comment]
Looks fine for me, except the code style ;)
Are you finished or is there something you want to add?

**Comment by SirPhoenix88** [comment]
I am finished with that part. Any other tweaks would likely take me a while.

**Comment by ThauEx** [comment]
Should I fix the codestyle for you? If so, I could merge it.

**Comment by SirPhoenix88** [comment]
Either that, or tell me how you would fix it so I know in case I contribute more.

**Comment by KHShadowrunner** [comment]
I'm actually not sure, seems that you used 2 space instead of tab, so that works. The only thing I see is line 142 is extended. Maybe you are using tabs and it just works a lot better than when I upload accidentally using tabs? either way, 2 space for every indent. Otherwise, it seems to mirror the one before.

**Comment by ThauEx** [comment]
Yes, two spaces is better.
Lines 137-141: Space after `*` and I would move the comment above the for loop.
Line 142: remove tab
Line 146: please add a blank line
Lines 147 - 151: Space after `*` and I would move the comment above the for loop.
Lines: 155-157: I think we can use a single line comment (`//`) here

File: lib/filter/buddy.js [hunk 1]
```diff
@@ -132,7 +133,29 @@ exports.update = function (json) {
...
+      /*
+       *Ability spammer
+       *Refills abilities per round and increases max to 10 (or decreases if 12, but it will refill every round)
+       *Fills soulbreak gauge one full unit per ability
+       */
+	    json[buddy].ability_panels[panel].num = '10';
+      json[buddy].ability_panels[panel].max_num = '10';
+      json[buddy].ability_panels[panel].ability_ss_point = '500'
+    }
+    for (var sStrike in json[buddy].soul_strikes) {
```
Comment (ThauEx) [inline lib/filter/buddy.js]: please add a blank line before

File: lib/filter/buddy.js [hunk 2]
```diff
@@ -132,7 +133,29 @@ exports.update = function (json) {
...
+      /*
+       *Soul strike spammer
+       *Allows all soul strikes to be instant-cast
+       *Makes them cost 1/500th of a soul gauge. (Normal cost is 500, new cost is one per soul break.)
+       */
+      json[buddy].soul_strikes[sStrike].options.cast_time = '100';
+      json[buddy].soul_strikes[sStrike].options.consume_soul_strike_point = '1';
+    }
+    /*
+     *Starts each new round with a full soul gauge
```
Comment (ThauEx) [inline lib/filter/buddy.js]: I think we can use a single line comment (//) here

File: lib/filter/buddy.js [hunk 3]
```diff
@@ -132,7 +133,29 @@ exports.update = function (json) {
...
       json[buddy].abilities[ability].options.cast_time = '100';
     }
-
+    for (var panel in json[buddy].ability_panels) {
+      /*
+       *Ability spammer
+       *Refills abilities per round and increases max to 10 (or decreases if 12, but it will refill every round)
+       *Fills soulbreak gauge one full unit per ability
+       */
+	    json[buddy].ability_panels[panel].num = '10';
```
Comment (ThauEx) [inline lib/filter/buddy.js]: remove tab

**Comment by SirPhoenix88** [comment]
Fixed, I think. Thanks for looking it over!

File: lib/filter/buddy.js [hunk 4]
```diff
@@ -132,7 +133,29 @@ exports.update = function (json) {
...
+     * Soul strike spammer
+     * Allows all soul strikes to be instant-cast
+     * Makes them cost 1/500th of a soul gauge. (Normal cost is 500, new cost is one per soul break.)
+     */
+    for (var sStrike in json[buddy].soul_strikes) {
+      json[buddy].soul_strikes[sStrike].options.cast_time = '100';
+      json[buddy].soul_strikes[sStrike].options.consume_soul_strike_point = '1';
+    }
+    
+    //Starts each new round with a full soul gauge
```
Comment (ThauEx) [inline lib/filter/buddy.js]: Just add a space after `//` and I'm happy

**Pull request merged**